### PR TITLE
Add Landing Page and Tests

### PR DESCRIPTION
**Pull request by iTitus**
The new images are:

----

Index:
shorty_index

----

Admin:
shorty_admin

### CODE REVIEW

**Review by Segelzwerg — APPROVED** [review]

**Comment by n2o** [comment]
Hm, i still like my landing page image, which is in the repository. Put into the browser makes it look like a real application and not just like a flat image. So please let my original image in the README and we do not need additional images for the Login Page.

File: README.adoc [hunk 1]
```diff
@@ -16,7 +16,12 @@ image::https://www.herokucdn.com/deploy/button.svg[link="https://heroku.com/depl
 This is a URL Shortener written in Java with the only purpose to create shortened urls, which are stored in a Redis-Database.
 The shortened links can then be used to provide a simpler access to complicated URLs.
 
-image::img/shorty.png[shorty]
```
Comment (n2o) [inline README.adoc:19]: Please leave the old image here

File: README.adoc [hunk 2]
```diff
@@ -16,7 +16,12 @@ image::https://www.herokucdn.com/deploy/button.svg[link="https://heroku.com/depl
...
+image::img/shorty_admin.png[shorty admin panel]
```
Comment (n2o) [inline README.adoc]:
```suggestion
```

File: README.adoc [hunk 3]
```diff
@@ -16,7 +16,12 @@ image::https://www.herokucdn.com/deploy/button.svg[link="https://heroku.com/depl
...
+
+It comes with authentication support. Only admins are allowed to create or delete short links.
+
+image::img/shorty_index.png[shorty index]
```
Comment (n2o) [inline README.adoc]:
```suggestion
```

**Review by n2o — CHANGES_REQUESTED** [review]
* Keep old image 
* Remove the new images

Thank you!

**Comment by n2o** [comment]
Please add in the next time a more descriptive title 😬 

This closes #19

**Comment by iTitus** [comment]
Can you make a new image of / and /admin?
Your browser ui looks better than mine

**Comment by n2o** [comment]
I don't see any improvement or better understanding when we add an image for each view :D the simplicity of one image is unbeatable und sufficient for the first impression of this app. I can make a new one for the Admin page, but the rest is not of interest.

**Comment by iTitus** [comment]
The image is outdated

**Comment by n2o** [comment]
Making a new image each time we change is string, is too much work for too little outcome.

**Comment by n2o** [comment]
shorty
here is a new Admin image

**Review by n2o — APPROVED** [review]

**Pull request merged**